server/dal/stock-info-old.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+const dotenv = require('dotenv');
+dotenv.config();
+
+var Schema = mongoose.Schema
+
+var stockInfoSchema = new Schema({
+    company_name: String,
+    ticker: String,
+    current_price: Number,
+    previous_close: Number,
+    open: Number,
+    last_price: Number,
```

Comment:
I think we decided not to use last_price as it's the same as previous_close. So we should update the schema to have only previous_close and remove last_price.